### PR TITLE
Responsive grid and add animation

### LINKED ISSUE #6: Create the layout of the first container of the landing page
Create the layout to display the website's 4 sections. That's the first thing the user will see when they land on the page. 
Each box will display the title of the section. The picture below is a rough draft of the section. 

landing page

### PR DESCRIPTION
**Pull request by sonianb**
Closes #6 

Made the grid responsive and added box shadow animation to grid elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,43 +37,49 @@ <h1>About</h1>
     </section>
 
     <section id="services">
-          <div class="flip-container">
```
Comment (sonianb) [inline index.html:40]: @PJSalter I formatted the document as well, that's why you see changes similar to this one - hope it's not confusing!

File: styles.css [hunk 2]
```diff
@@ -72,9 +78,13 @@
 
 #services-grid {
     grid-area: services;
-    font-size: 3rem;
     color: var(--secondary-color);
     border: var(--secondary-color) solid 20px;
+    text-orientation: mixed;
```
Comment (PJSalter) [inline styles.css:83]: I would say that this CSS property is a very valid option in changing the direction of the text, makes the text pleasing toward the eye once viewed 🙌

File: styles.css [hunk 3]
```diff
@@ -87,10 +97,14 @@
 }
 
 .grid-item {
-    border: solid black 1px;
     text-align: center;
     font-size: 3.5rem;
 }
+.grid-item:hover {
+    transition: transform 100ms ease-in-out;
```
Comment (PJSalter) [inline styles.css:104]: a reasonable time duration  for motion effect of hover 🌟 Also box shadowing is a good feature to add for making the content pop out 😸 I use it myself, great work ✨

File: styles.css [hunk 4]
```diff
@@ -174,14 +188,14 @@ h2, h3 {
     .grid-container {
         display: flex;
         flex-flow: column wrap;
-        max-width: 15rem;
         margin-left: auto;
         margin-right: auto;
         align-content: center;
+        row-gap: 0.3rem;
```
Comment (PJSalter) [inline styles.css:194]: Good amount of spacing from row gap, keeps all the boxes neat 😄

File: styles.css [hunk 5]
```diff
@@ -59,6 +63,8 @@
     align-items: flex-end;
     color: var(--secondary-color);
     border: var(--secondary-color) 20px solid;
+    text-orientation: mixed;
+    writing-mode: sideways-lr;
```
Comment (PJSalter) [inline styles.css:67]: Perfect flow for text direction on text

**Review by PJSalter — APPROVED** [review]
Theres some brilliant improvements in animation effects as well as adaptation in styling, keeps it all consistent, really good work ⭐

File: index.html (hunk 1 above)
Comment (PJSalter) [reply index.html:40]: It's neater now so it's all good, i read through the changes. Thank You

**Pull request merged**